releng/tools/rmake

```diff
@@ -336,6 +337,17 @@ def tidy(patterns):
         sys.exit(1)
 
 def main():
+    maxThreads = 2 + multiprocessing.cpu_count()
+    for i in range(len(sys.argv)):
+        if sys.argv[i][:2] == "-j":
+            if not sys.argv[i][2:].isdecimal() or round(float(sys.argv[i][2:])) <= 0:
+                sys.exit("Invalid number of threads to use")
+            
+            maxThreads = min(round(float(sys.argv[i][2:])), maxThreads)
```

Comment:
I believe that float should always be invalid

Reply:
The float can be valid for "-4" or strings as such, but those are filtered out by the first statement in the condition, so technically the second statement could be just `round(float(sys.argv[i][2:])) != 0` (I assume -j0 passed to the compiler should not be valid, so I took care of it in advance here)

Reply:
Well, my question is why it is parsing a float and rounding? Why don't you just parse an int?

Reply:
Also - you shouldn't cap the number of threads by the maxThreads - there could be situations where the user wants to use a larger number - but most importantly you shouldn't silently change the user request.

Reply:
I do not remember or even know why I used float() in the first place; you are of course correct that it works with just parsing an int.
I have changed it now, and removed the maxThreads cap.